cartography/intel/aws/cloudtrail_management_events.py

```diff
@@ -0,0 +1,553 @@
+    return assumed_role_arn
+
+
+@timeit
+def sync(
+    neo4j_session: neo4j.Session,
+    boto3_session: boto3.Session,
+    regions: List[str],
+    current_aws_account_id: str,
+    update_tag: int,
+    common_job_parameters: Dict[str, Any],
+) -> None:
+    """
+    Sync CloudTrail management events to create ASSUMED_ROLE relationships.
+
+    This function orchestrates the complete process:
+    1. Fetch CloudTrail management events from all regions
+    2. Transform events into role assumption records
+    3. Load aggregated role assumption relationships into Neo4j
+
+    The resulting graph contains direct relationships like:
+    (AWSUser|AWSRole|AWSPrincipal)-[:ASSUMED_ROLE {times_used, first_seen, last_seen, lastused}]->(AWSRole)
+
+    :type neo4j_session: neo4j.Session
+    :param neo4j_session: The Neo4j session
+    :type boto3_session: boto3.Session
+    :param boto3_session: The boto3 session to use for API calls
+    :type regions: List[str]
+    :param regions: List of AWS regions to sync
+    :type current_aws_account_id: str
+    :param current_aws_account_id: The AWS account ID being synced
+    :type aws_update_tag: int
+    :param aws_update_tag: Timestamp tag for tracking data freshness
+    :rtype: None
+    """
+    # Extract lookback hours from common_job_parameters (set by CLI parameter)
+    lookback_hours = common_job_parameters.get(
+        "aws_cloudtrail_management_events_lookback_hours"
+    )
+
+    if not lookback_hours:
+        logger.info(
+            "CloudTrail management events sync skipped - no lookback period specified"
+        )
+        return
+
+    logger.info(
+        f"Starting CloudTrail management events sync for account {current_aws_account_id}"
+    )
+    logger.info(
+        f"Syncing {len(regions)} regions with {lookback_hours} hour lookback period"
+    )
+
+    all_role_assumptions = []
+
+    # Fetch events from all regions
+    for region in regions:
+        try:
+            # Get raw CloudTrail events
+            events = get_cloudtrail_events(
+                boto3_session=boto3_session,
+                region=region,
+                lookback_hours=lookback_hours,
+            )
+
+            # Transform to role assumptions
+            role_assumptions = transform_cloudtrail_events_to_role_assumptions(
+                events=events,
+                region=region,
+                current_aws_account_id=current_aws_account_id,
+            )
+
+            all_role_assumptions.extend(role_assumptions)
+
+        except Exception as e:
+            logger.warning(
+                f"Failed to process CloudTrail events for region {region}: {str(e)}"
+            )
+            continue
+
+    # Load all role assumptions into Neo4j and run cleanup
+    if all_role_assumptions:
+        load_role_assumptions(
+            neo4j_session=neo4j_session,
+            role_assumptions=all_role_assumptions,
+            region="all",  # Indicates cross-region aggregation
```

Comment:
What is `'all'`? There is no such thing as an `'all'` region. Remove this param.